src/DocumentFormat.OpenXml/Packaging/OpenXmlPackage.FlatOpc.cs

```diff
@@ -130,11 +130,26 @@ private static XElement GetBinaryPartContentsAsXml(PackagePart part)
         private static string ToChunkedBase64String(PackagePart part)
         {
             using var stream = part.GetStream();
-            using var memory = new MemoryStream((int)stream.Length);
+            var size = checked((int)stream.Length);
+            var buffer = new byte[size];
```

Comment:
if you're going for perf, would you like to update this to use `ArrayPool<byte>` here as well? Then you'd not need to allocate on platforms that supports that.

NOTE: we're currently not shipping a .NET 6+ build, but I'm fine with having the #if/#defs for perf stuff and we can decide to turn a build for that on fairly easily.

Reply:
> if you're going for perf, would you like to update this to use ArrayPool<byte> here as well? Then you'd not need to allocate on platforms that supports that.

FlatOpc is not a very popular variation of OOXML, so I'm OK with the current implementation. This PR was intended to reduce memory footprint only 😄 

> NOTE: we're currently not shipping a .NET 6+ build, but I'm fine with having the #if/#defs for perf stuff and we can decide to turn a build for that on fairly easily.

I've added if/def just in order not to forget in the future that there is an existing method for that.